### DISCUSSION TITLE
Is there any easy way to convert Eb/No to SNR in Sionna?

### DISCUSSION THREAD

**Discussion by Mio-TW** [NVlabs/sionna#298] [Q&A]
Hi everyone,

I want to plot the BLER vs. SNR figure in this [example](https://nvlabs.github.io/sionna/examples/Neural_Receiver.html).
But I'm not sure if there is any way to convert Eb/No to SNR function more easily?

If I need to do the calculations myself, are the conversions below correct?
(assume Eb/No=5dB, 64QAM modulation is used)
SNR = 5dB + log2(64) = 11dB 

Thank you so much!

**Comment by jhoydis** [comment]
Hi,

The documentation of the function [``ebnodb2no``](https://nvlabs.github.io/sionna/api/utils.html?highlight=ebnodb2no#sionna.utils.ebnodb2no) tells you what you need to do. Simply solve the equation for `E_s/N_o`.

In your calculation above, you might want to include the coderate.
